<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>ICU</string>
     </attribute>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Code</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Manager</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Iname_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="IManager_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Iadd_pushButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>180</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Capacity</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Icode_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Icapacity_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Idisplay_pushButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>270</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Display ICU List</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ass_lineEdit">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>34</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ass_pushButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>70</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Assign Patient</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>New Patient</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Birth Date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Medical History</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Allergies</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Blood Pressure</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Heart Rate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>450</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Severity</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Pmedical_history_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Psex_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Pallergies_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>290</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Pbirthday_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Pblood_pressure_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>330</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Pheart_rate_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>370</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Ptemp_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>410</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Pseverity_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>450</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Pphone_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Paddress_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Pname_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Padd_pushButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>160</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="counter_pushButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>230</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Counter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="display_pushButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>310</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Display Patient List</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>New Doctor</string>
     </attribute>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Major</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Degree</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Dbirthday_lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DDegree_lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Dname_lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="DMajor_lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Birth Date</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Dsex_lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Dadd_pushButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>190</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ddisplay_pushButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>280</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Display Doctor List</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>857</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>Iname_lineEdit</tabstop>
  <tabstop>IManager_lineEdit</tabstop>
  <tabstop>Icode_lineEdit</tabstop>
  <tabstop>Icapacity_lineEdit</tabstop>
  <tabstop>Iadd_pushButton</tabstop>
  <tabstop>Idisplay_pushButton</tabstop>
  <tabstop>ass_lineEdit</tabstop>
  <tabstop>ass_pushButton</tabstop>
  <tabstop>Pname_lineEdit</tabstop>
  <tabstop>Paddress_lineEdit</tabstop>
  <tabstop>Pphone_lineEdit</tabstop>
  <tabstop>Pbirthday_lineEdit</tabstop>
  <tabstop>Psex_lineEdit</tabstop>
  <tabstop>Pmedical_history_lineEdit</tabstop>
  <tabstop>Pallergies_lineEdit</tabstop>
  <tabstop>Pblood_pressure_lineEdit</tabstop>
  <tabstop>Pheart_rate_lineEdit</tabstop>
  <tabstop>Ptemp_lineEdit</tabstop>
  <tabstop>Pseverity_lineEdit</tabstop>
  <tabstop>Padd_pushButton</tabstop>
  <tabstop>counter_pushButton</tabstop>
  <tabstop>display_pushButton</tabstop>
  <tabstop>Dname_lineEdit</tabstop>
  <tabstop>DMajor_lineEdit</tabstop>
  <tabstop>DDegree_lineEdit</tabstop>
  <tabstop>Dbirthday_lineEdit</tabstop>
  <tabstop>Dsex_lineEdit</tabstop>
  <tabstop>Dadd_pushButton</tabstop>
  <tabstop>Ddisplay_pushButton</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
